views/index.handlebars

```diff
@@ -50,17 +51,33 @@
 		updatePublishStatus(msg);
 	});
 	socket.on('publish-failure', function(msg){
-		document.getElementById('publish-active-area').innerHTML = '<p> --(✖╭╮✖)→ </p><p>' + JSON.stringify(msg) + '</p><strong>For help, post the above error text in the #speech channel on the <a href="https://lbry.slack.com/" target="_blank">lbry slack</a></strong>';
+		document.getElementById('publish-active-area').innerHTML = '<p> --(✖╭╮✖)→ </p><p>' + JSON.stringify(msg) + '</p><strong>For help, post the above error text in the #speech channel on the <a class="link--primary" href="https://lbry.slack.com/" target="_blank">lbry slack</a></strong>';
 	});
 	socket.on('publish-complete', function(msg){
 		var publishResults;
-		var showUrl = msg.name + '/' + msg.result.claim_id;
+		var showUrl = msg.result.claim_id + '/' + msg.name;
 		// build new publish area
 		publishResults = '<p>Your publish is complete! You are being redirected to it now.</p>';
-		publishResults += '<p><a target="_blank" href="' + showUrl + '">If you do not get redirected, click here.</a></p>';
+		publishResults += '<p><a class="link--primary" target="_blank" href="' + showUrl + '">If you do not get redirected, click here.</a></p>';
 		// update publish area
 		document.getElementById('publish-active-area').innerHTML = publishResults;
+		// store the publish in local storage
+        const myStorage = window.localStorage;
```

Comment:
Storing this data server side and using a key to recover it would give you more flexibility in the data structure.

As-is, if this data structure changes, we'd have to either write client-side migrations or invalidate all client-side data.

One option is to store a token unique to publisher (user) on each entry in the file table. Knowing this token could grant edit rights in addition to acting as a retrieval filter.